Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -377,17 +363,9 @@ - (void)logIn
 
 - (void)logOut
 {
-  void (^afterDeauthorization)() = ^() {
-    
-    [[NYPLMyBooksDownloadCenter sharedDownloadCenter] reset:self.accountType];
-    [[NYPLBookRegistry sharedRegistry] reset:self.accountType];
-    
-    [[NYPLAccount sharedAccount:self.accountType] removeAll];
-    [self setupTableData];
-    [self.tableView reloadData];
-  };
   
 #if defined(FEATURE_DRM_CONNECTOR)
+  
```

Comment:
This is a really minor thing, but it's possible to configure Xcode so it won't insert spaces on blank lines like this and clutter up diffs. Try setting the following in Preferences > Text Editing > Editing:

![screen shot 2017-07-11 at 7 35 29 pm](https://user-images.githubusercontent.com/10768108/28095389-1ffd8b6a-6670-11e7-9174-46038a8bbf70.png)